pkg/asset/installconfig/powervs/platform.go

```diff
@@ -0,0 +1,15 @@
+package powervs
+
+import "fmt"
+
+func ValidateAccountPermissions(client *Client) (error) {
+	accType, err := client.GetAccountType()
+	if err != nil {
+		return err
+	}
+
+	if accType == "TRIAL" {
+		return fmt.Errorf("Provided account is Trial account")
```

Comment:
is that the equivalent of "lite" that I saw in the documentation?

Reply:
Yes.. It is equivalent to lite